controller/villains.js

```diff
@@ -0,0 +1,46 @@
+const models = require('../models')
+
+const getAllVillains = async (request, response) => {
+  try {
+    const villainsData = await models.Villains.findAll()
+
+    return response.send(villainsData)
+  } catch (error) {
+    return response.status(500).send('Unable to retrieve villain list, try again')
+  }
+}
+
+const getVillainBySlug = async (request, response) => {
+  try {
+    const { slug } = request.params
+
+    const foundVillain = await models.Villains.findOne({ where: { slug: slug } })
+
```

Comment:
Use attribute property to control what is viewed

const pickedVillain = await models.villains.findOne({ where: { slug }, attributes: ['name', 'movie', 'slug'], })